src/components/Tool/Home/Home.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ReactElement, useState } from 'react';
+import { makeStyles, Theme, Grid, Container, Box } from '@material-ui/core';
+import { motion } from 'framer-motion';
+
+import NavToolDrawer from '../NavToolList/NavToolDrawer';
+import Tool from './Tool';
+import { ToolsList } from './ToolList';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  container: {
+    padding: 20,
+    marginRight: 17,
+    display: 'flex',
+    [theme.breakpoints.down('lg')]: {
+      paddingLeft: 30,
+      marginRight: 20,
+    },
+    [theme.breakpoints.down('md')]: {
+      paddingLeft: 40,
+      marginRight: 15,
+    },
+    [theme.breakpoints.down('sm')]: {
+      paddingLeft: 45,
+    },
+  },
+  toolIcon: {
+    height: '40px',
+    width: '40px',
+    color: theme.palette.cruGrayDark.main,
+  },
```

Comment:
Can you use the [spacing helper](https://material-ui.com/customization/spacing/#spacing) for any of these magic numbers? It's ok if you need a specific value, just having some consistency and fewer magic numbers is nice.

Reply:
Changed